### ISSUE TITLE
Change Learning Rate

### ISSUE THREAD

**Issue by bwliv** [AmoghM/Reinforcement-Learning-Trade-Strategies#52]
This is the hyperparameter tuning that matters
